### PR TITLE
Update Map Styles

### PR DESCRIPTION
**Pull request by nickspeal**
Give the look and feel of the map a bit of a refresh:

- Update the styles in Mapbox Studio and point to the new styleset ID
  - Update colors of many lines and regions
  - Update fonts
- Update the colors for the route overlays
- Replace the map icons with new SVGs
- Update the min zoom level to prevent zooming out too far away from Tahoe
- Turn plowed winter paths back on by default as snow is coming!
- Don't load the map with the `mapbox.streets` layer and then a 100% opacity tile layer on top. This should at least halve the amount of data requested.
- Also, npm update, because why not

https://tahoebike-stage.herokuapp.com/

Screen Shot 2019-11-24 at 10 31 59 PM

Screen Shot 2019-11-24 at 10 33 33 PM

### CODE REVIEW

**Comment by nickspeal** [comment]
I'll merge this pending feedback from the LTBC board

File: public/js/map.js [hunk 1]
```diff
@@ -255,12 +256,10 @@ exports.drawMap = (center, zoom, minZoom, draggable, handleMapClick, handleMarke
   createConstructionLayer();
 
   class1Layer.addTo(map);
-  class1LayerOutline.addTo(map);
   class2Layer.addTo(map);
   class3Layer.addTo(map);
   constructionLayer.addTo(map);
-  // Uncomment to add winterlayer by default:
-  // winterLayer.addTo(map);
+  winterLayer.addTo(map);
```
Comment (brendannee) [inline public/js/map.js:271]: You could do this programmatically by date if you know you always want it to show up on a certain date and disappear after a certain date.

**Review by brendannee — APPROVED** [review]
Great new styles and updates.

**Comment by nickspeal** [comment]
Thanks for taking a look Brendan 👍

File: public/js/map.js (hunk 1 above)
Comment (nickspeal) [reply public/js/map.js:271]: ooh good call

**Pull request merged**